gemfirexd/shared/src/main/resources/com/pivotal/gemfirexd/internal/loc/toolsmessages.properties

```diff
@@ -545,6 +569,86 @@ FS_EXTRA_HELP=\
 \                 data and will act as an accessor JVM still having the\n\
 \                 capability to route queries to appropriate datastores and\n\
 \                 aggregating the results.\n
+#snappy specific message hides mcast port args, otherwise same as above
+FS_SNAPPY_EXTRA_HELP=\
+\   -sync         If true then wait for complete initialization of the server\n\
+\                 including DataDictionary, persistent tables, network server,\n\
+\                 else return when this node starts waiting for other members\n\
+\                 to join to sync persistent data (default false).\n\
+\   -heap-size    Sets both the initial heap and max heap sizes.\n\
+\                 Size has the same format as the -Xmx/-Xms JVM options.\n\
+\                 -heap-size is equivalent to specifying both -J-Xmx and -J-Xms.\n\
+\   -off-heap-size Size has the same format as the -Xmx/-Xms JVM options.\n\
+\   -locators     The list of locators as comma-separated host:port values used\n\
+\                 to communicate with running locators in the system to\n\
+\                 discover other peers of the distributed system. The list\n\
+\                 must include all locators in use, and must be configured\n\
+\                 consistently for every member of the distributed system.\n\
+\   -start-locator Specified an embedded locator in this server that will be\n\
```

Comment:
Can remove this from normal help output. Will unnecessarily confuse users and no one will use it except in extremely specialized cases.